package.json

```diff
@@ -64,10 +64,10 @@
     "@types/lodash.escaperegexp": "^4.1.7",
```

Comment:
Better not to add too many unrelated changes or they will make a mess of git's history.